scripts/cadence_conformal/README.md

```diff
@@ -0,0 +1,25 @@
+# Logic Equivalence Checking (LEC)
+
+This folder contains a LEC script that runs on 
+Cadence Design Systems CONFORMAL.
+
+The current scripts have been tried on version 20.10 on a 64 bit executable.
+
+The `cv32e40p_lec_cmp.csh` runs on C-shell.
+
+Before executing it, please define the GOLDEN_RTL and REVISED_RTL 
+paths as enviromental variables.
+
+
+```
+setenv GOLDEN_RTL=YOUR_GOLDEN_CORE_RTL_PATH
```

Comment:
No '=' sign I believe (same for next line)

Reply:
true sorry